data/mods/More_Survival_Tools/recipes.json

```diff
@@ -614,26 +614,16 @@
   ]
 },{
   "type" : "recipe",
-  "override": true,
   "result": "plant_fibre",
+  "override": true,
```

Comment:
Merely restoring how it was before I copied the mainline version of the recipe.

Overrides are no longer required for mod recipes? If so, might be worth removing them from recipes in a future pull request. But as with the Arcana mod fixes, I would rather focus on getting them fixed first, and adding less-essential changes afterward.